Makefile

```diff
@@ -1,12 +1,14 @@
 abuild:
-	 docker-compose run --rm app su builder -c "cd /home/builder/aports/$(PKG) && abuild -f -r"
+	docker-compose run --rm app sh -c \
+	  "cd /home/builder/aports/$(PKG) && abuild -r"
```

Comment:
Nice catch removing the leading space after the tab. Could you please mention why the `-f` was removed? I don't remember why it was added, as the official docs - which this project is based on - [doesn't suggests it][1], but I'm worried that it was there because of an edge case that might re-appear in the future.

[1]: https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#Build_the_package

Reply:
I think it was removed simply because the official docs don't suggest it. From a quick look at the code, it seems that `-f` simply forces rebuild regardless of whether or not the currently build APK is up-to-date. Maybe if I add a second target that forces rebuild? (Always forcing it seems a bit excessive to me :)

Reply:
I agree, but no need for a forced target. If I ever face problems with that again I'll take care of properly documenting the reason around the decision.